pkg/api/customization/namespace/namespace.go

```diff
@@ -93,7 +93,8 @@ func NewFormatter(next types.Formatter) types.Formatter {
 			next(request, resource)
 		}
 		annotations := convert.ToMapInterface(resource.Values["annotations"])
-		if convert.ToString(annotations[helm.AppIDsLabel]) == "" {
+		_, hasQuota := resource.Values["resourceQuota"]
+		if convert.ToString(annotations[helm.AppIDsLabel]) == "" && !hasQuota {
```

Comment:
we should check the resourceQuota presence field on the project you are moving to, not on the namespace